systemd/ua-reboot-cmds.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Ubuntu Advantage reboot cmds
+
+[Service]
+Type=oneshot
+ConditionPathExists=/var/lib/ubuntu-advantage/marker-reboot-cmds-required
```

Comment:
I think this needs to be in the [Unit] section, othewise we see this message in `journalctl -b` 
` [/lib/systemd/system/ua-reboot-cmds.service:6] Unknown lvalue 'ConditionPathExists' in section 'Service''